samples/wifi/shell/README.rst

```diff
@@ -0,0 +1,147 @@
+.. _wifi_shell:
+
+Wi-Fi: Shell
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample allows testing Nordic Semiconductor's Wi-Fi chipsets.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+This sample can perform all Wi-Fi operations in 2.4GHz and 5GHz bands depending on the capabilities supported in the underlying chipset.
+
+Using this sample, the development kit can associate with, and ping to, any Wi-Fi capable access point in :abbr:`STA (Station)` mode.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/shell`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Currently, the following configurations are supported:
+
+* 7002 DK + QSPI
+* 7002 EK + SPIM
+
+
+To build for the nRF7002 DK, use the ``nrf7002dk_nrf5340_cpuapp`` build target.
+The following is an example of the CLI command::
+
+   west build -b nrf7002dk_nrf5340_cpuapp
+
+To build for the nRF7002 EK, use the ``nrf7002dk_nrf5340_cpuapp`` build target with the ``SHIELD`` CMake option set to ``nrf7002_ek``.
+The following is an example of the CLI command::
+
+   west build -b nrf5340dk_nrf5340_cpuapp -- -DSHIELD=nrf7002_ek
+
+See also :ref:`cmake_options` for instructions on how to provide CMake optio
+
+Testing
+=======
+
+|test_sample|
+
+#. |connect_kit|
+#. |connect_terminal|
+#. Scan for the Wi-FI networks using the following command::
+
+      uart:~$ wpa_cli scan
+
+   The output should be similar to the following::
+
+      Scan requested
+      Num  | SSID                             (len) | Chan | RSSI | Sec
+      1    | dlink-7D64                       10    | 1    | 0    | WPA/WPA2
+      2    | abcd_527_24                      11    | 4    | 0    | Open
+      3    | ASUS_RTAX88U11g                  15    | 3    | 0    | WPA/WPA2
+      4    | TP-Link_6BA8                     12    | 9    | 0    | WPA/WPA2
+
+#. Connect to a preferred network using the following command::
+
+      uart:~$ wpa_cli connect <SSID> <passphrase>
+
+   ``<SSID>`` is the SSID of the network you want to connect to, and ``<passphrase>`` is its passphrase.
+
+#. Check the connection status after a while, using the following command::
+
+      uart:~$ wpa_cli status
+
+   If the connection is established, you should see an output similar to the following::
+
+      Reply: bssid=c0:06:c3:1d:cf:9e
+      freq=0
+      ssid=ASUS_RTAX88U11g
+      id=0
+      mode=station
+      pairwise_cipher=CCMP
+      group_cipher=CCMP
+      key_mgmt=WPA2-PSK
+      pmf=1
+      mgmt_group_cipher=BIP
+      wpa_state=COMPLETED
+      ip_address=192.168.0.206
+      address=f4:ce:36:00:00:16
+
+#. Initiate a ping and verify data connectivity using the following command.
+
```

Comment:
The point here was to highlight the parameter in the subcommand in a generic way instead of using "google.com". Feel free to use one that you believe is the most technically correct.